### PR TITLE
Changes for DEV-68228 and DEV-68229

### PR DESCRIPTION
**Pull request by gagansingh355**

### CODE REVIEW

**Comment by ahsan13jan** [comment]
@gagansingh355 You were not able to find a template arn for network interface?

File: code/parse.go [hunk 1]
```diff
@@ -99,6 +101,10 @@ func parseCloudWatchLogs(request events.CloudwatchLogsEvent) []ingest.Log {
 		result := re1.FindStringSubmatch(d.LogGroup)
 		lambdaName := result[1]
 		arn = fmt.Sprintf("arn:aws:lambda:%s:%s:function:%s", awsRegion, d.Owner, lambdaName)
+	} else if strings.Contains(d.LogGroup, "/aws/ec2/networkInterface") {
+		splitLogStream := strings.Split(d.LogStream, "-")
+		arn = splitLogStream[0] + "-" + splitLogStream[1]
+		resourceID = "system.aws.networkInterfaceIds"
```
Comment (ahsan13jan) [inline code/parse.go:107]: If we are not sending the arn for network interface. Maybe we should rename the variable to something else.
I am curious, if we could still do it with arn. AWS should be able to identify that.

**Review by ahsan13jan — CHANGES_REQUESTED** [review]
If we are not sending the arn for network interface. Maybe we should rename the variable to something else.
I am curious, if we could still do it with arn. AWS should be able to identify that.

File: code/parse.go (hunk 1 above)
Comment (gagansingh355) [reply code/parse.go:107]: I don't think we would be able to do with ARN as EC2 recognises network interface through a property. Please refer to comment section of https://jira.logicmonitor.com/browse/DEV-68228.

I will update the variable name.

**Pull request closed without merging**